adhocracy-plus/config/settings/base.py

```diff
@@ -36,7 +36,6 @@
     "easy_thumbnails",
     "ckeditor",
     "ckeditor_uploader",
-    "background_task",
     "parler",
```

Comment:
oops this needs to go back for the time being.

Reply:
good catch, done.